rts/Game/UI/MiniMap.cpp

```diff
@@ -1088,6 +1103,12 @@ void CMiniMap::Update()
 	fbo.Bind();
 	UpdateTextureCache();
 
+	if (curPos != lastPos || curDim != lastDim) { // probably can be moved to SetGeometry
```

Comment:
Can we move this to `SetGeometry` or is there some concern with geometry state being changed in ways we can't track?

Reply:
The entire file continuously changes dim and pos from curDim and curPos regularly from many functions, a simple swap and replace doesn't work either cuz some lines require the actual curX

TLDR: still seeing if we can do so

Reply:
Sounds good! I'll mark this as resolved as I can't offer an obvious straightforward solution that does not make the callin spammy (invoked multiple times for situations it should be condensed in a single one)

Reply:
If it doesn't work out consider
```suggestion
	/* Would need a larger refactor for this to become SetGeometry since the entire file
	 * continuously changes dim and pos from curDim and curPos regularly from many functions */
	if (curPos != lastPos || curDim != lastDim) {
```